pubspec.yaml

```diff
@@ -17,6 +17,8 @@ dependencies:
   # Use with the CupertinoIcons class for iOS style icons.
   cupertino_icons: ^0.1.2
 
+  intl: 0.15.7
```

Comment:
write ^0.15.7 instead of 0.15.7 as you don't need precisely 0.15.7

Reply:
Done.